CHANGELOG.txt

```diff
@@ -35,6 +35,10 @@
     # Apps :
         * Activities :
             - The JavaScript component FullCalendar has been upgraded to the version "6.1.15"
+            - Improve CSS of the calendar sidebar and make it sticky.
+            - New 'My activities calendar' brick is now available for the home page-view.
+            - New calendar settings for the visible hours range.
+            - In the calendar configuration, the visible hours range is added.
```

Comment:
- Duplicate
- what is visible hours range? what's the difference with day_start/day_end?

Reply:
day_start/end is related to "working hours". view_day_start/end is for the "displayed hours".